portfolio/src/main/webapp/script.js

```diff
@@ -77,35 +80,47 @@ function updateComments(comments, element) {
 /** Creates an <li> element containing text. */
 function createListElement(comment) {
     const liElement = document.createElement('li');
+    liElement.setAttribute("class", "comment");
     var rounded_score = comment.sentiment_score.toFixed(2);
 
-    liElement.innerHTML = comment.content + '<br><br><div class="comment-data">Posted by ' + 
+    // const titleElement = document.createElement('span');
+    liElement.innerHTML = comment.content + '<br><br>Posted by ' + 
         comment.user_name + ' (' + comment.user_location + ') at ' + comment.formatted_time +
-        '<br>Sentiment score: ' + rounded_score + '</div>';
+        '<br>Sentiment score: ' + rounded_score;
 
-    liElement.setAttribute("class", "comment");
     setSentimentColor(liElement, comment.sentiment_score);
+
+    const deleteButtonElement = document.createElement('button');
+    deleteButtonElement.classList.add("delete-comment");
+    deleteButtonElement.innerHTML = "<i class='fa fa-remove'></i>";
```

Comment:
Tricky font awesome work.
It was a bit of a puzzle to work out, not initially knowing about that library.

It still seems odd to me that the button would use a `<i>` tag for this rather than, say `<span>`. But that does seem to be the convention (probably because it is short and may suggest "icon" as well as "italics". In any case, it is best to follow conventional usage, as that is what everyone expects.

Do you happen to know how it works?